packages/core/src/InternalLog.tsx

```diff
@@ -6,6 +6,11 @@
 
 import { SdkVerbosity } from './SdkVerbosity';
 
+/**
+ * /!\ DO NOT USE THIS IN YOUR APP /!\\
```

Comment:
I have to use an extra `\` at the end of the line so that it is displayed nicely in IDEs.